app/src/main/java/com/poupa/vinylmusicplayer/discog/tagging/ReplayGainTagExtractor.java

```diff
@@ -119,7 +140,18 @@ private static Map<String, Float> parseLameHeader(@NonNull final AudioFile file)
     String lameMark = new String(chunk, 0, 4, "ISO-8859-1");
 
     if (lameMark.equals("Info") || lameMark.equals("Xing")) {
-      s.seek(0xAB);
```

Comment:
Just to make sure that I understand: Instead going directly to offset `0xAB` we consume the 4 bytes right before that, at  offset `0xA7`, then continue the parsing business like before?

Reply:
Yes, in a comment, I've added a link to a web page that indicates that the peak is stored as a 32b float before gain info. It seems that format also stores only 1 peak value, unlike the other formats.